jiwonjang/app.js

```diff
@@ -29,14 +29,34 @@ app.use(express.json());
 app.use(cors());
 app.use(morgan("dev"));
 
+const PORT = process.env.PORT;
+const HOST = process.env.HOST;
+
 //health check
 app.get("/ping", (req, res) => {
   return res.status(200).json({ message: "pong" });
 });
 
-const PORT = process.env.PORT;
+//create users
+
+app.post("/users", async (req, res) => {
+  const { name, email, profileImage, password } = req.body;
+
+  //console.log(req)
+
+  await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      profile_image,
+      password
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [name, email, profileImage, password]
+  );
+  res.status(201).json({ message: "useCreated" });
```

Comment:
```suggestion
  return res.status(201).json({ message: "useCreated" });
```
실행문의 끝을 확실하게 하기 위해 return문을 작성해주셔도 좋을 것 같습니다.